bin/license/license_setup.sh

```
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set +eu
+
+source nextflow-bin/license/helper_funcs.sh
+SENTIEON_INSTALL_DIR=$(sentieon_install_dir)
+sentieon_procs=$(nproc)
+# Setup license
+# find corresponding project where license token information is and download it
+find_license_project
+if [ "$license_project" == "" ]; then
```

Shellcheck (0.10.0)

[warning] license_project is referenced but not assigned.

(SC2154)